rust/relay/ebpf-turn-router/src/error.rs

```diff
@@ -1,8 +1,46 @@
 use core::num::NonZeroUsize;
 
+// Helper function to map Linux/eBPF error codes to human-readable strings
```

Comment:
I don't feel strongly about having these. They did prevent one verifier issue at one point and could help the (likely frustrated) engineer who works on this next.